wiki/en/Running-a-Server.md

```diff
@@ -216,15 +164,6 @@ function loadserverstat ( $statfilename )
 ~~~
 
 
-# Servers on the desktop
-
-Jamulus can be configured to run in Server mode on the desktop. This gives you a graphical user interface to control most of the settings. 
-
-* **Windows users** - Use the "Jamulus Server" icon in the Windows Start menu.
-
-* **macOS users** - Double-click the "Jamulus Server" icon in Applications (assuming you put the files from the install there as per [these instructions](Installation-for-Macintosh)).
-
-* **Linux users** - Launch the "Jamulus Server" shortcut. Or you can open a terminal window (`CTRL+ALT+t` on Debian and related distros), type `jamulus -s` and hit return. _(To run a headless Server [read this guide](#installing-and-running-a-server))_
 
 
 ## Server Setup
```

Comment:
There is a section called "Setting up a server" further up the page. This all now seems a bit confusing.